src/Microsoft.ApiDesignGuidelines.Analyzers/Core/OperatorOverloadsHaveNamedAlternates.Fixer.cs

```diff
@@ -26,11 +28,130 @@ public sealed override FixAllProvider GetFixAllProvider()
+                    var typeDeclaration = generator.GetDeclaration(typeDeclarationSyntax, DeclarationKind.Class);
+
+                    SyntaxNode addedMember;
+                    var bodyStatements = ImmutableArray.Create(
+                        generator.ThrowStatement(generator.ObjectCreationExpression(semanticModel.Compilation.GetTypeByMetadataName("System.NotImplementedException"))));
+                    if (OperatorOverloadsHaveNamedAlternatesAnalyzer.IsPropertyExpected(operatorOverloadSymbol.Name))
+                    {
+                        // add a property
+                        addedMember = generator.PropertyDeclaration(
+                            name: OperatorOverloadsHaveNamedAlternatesAnalyzer.IsTrueText,
+                            type: generator.TypeExpression(SpecialType.System_Boolean),
+                            accessibility: Accessibility.Public,
+                            modifiers: DeclarationModifiers.ReadOnly,
+                            getAccessorStatements: bodyStatements);
+                    }
+                    else
+                    {
+                        // add a method
+                        var expectedSignature = GetExpectedMethodSignature(operatorOverloadSymbol, semanticModel.Compilation);
+                        addedMember = generator.MethodDeclaration(
+                            name: expectedSignature.Name,
+                            parameters: expectedSignature.Parameters.Select(p => generator.ParameterDeclaration(p.Item1, generator.TypeExpression(p.Item2))),
+                            returnType: generator.TypeExpression(expectedSignature.ReturnType),
+                            accessibility: Accessibility.Public,
+                            modifiers: expectedSignature.IsStatic ? DeclarationModifiers.Static : DeclarationModifiers.None,
+                            statements: bodyStatements);
+                    }
+
+                    var newTypeDeclaration = generator.AddMembers(typeDeclaration, addedMember);
+                    return context.Document.WithSyntaxRoot(root.ReplaceNode(typeDeclaration, newTypeDeclaration));
+                case OperatorOverloadsHaveNamedAlternatesAnalyzer.FixVisibilityText:
+                    var badVisibilityNode = generator.GetDeclaration(node, DeclarationKind.Method) ?? generator.GetDeclaration(node, DeclarationKind.Property);
+                    var badVisibilitySymbol = semanticModel.GetDeclaredSymbol(badVisibilityNode, context.CancellationToken);
+                    var symbolEditor = SymbolEditor.Create(context.Document);
+                    var newSymbol = await symbolEditor.EditOneDeclarationAsync(badVisibilitySymbol,
+                        (documentEditor, syntaxNode, ct) => Task.Run(() => documentEditor.SetAccessibility(badVisibilityNode, Accessibility.Public))).ConfigureAwait(false);
```

Comment:
same as aboive. I think you should do return Task.FromResult... rather than asking it to actually run task.